deps/chakrashim/lib/chakra_inspector.js

```diff
@@ -0,0 +1,271 @@
+// Copyright Microsoft. All rights reserved.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the 'Software'), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and / or
```

Comment:
I don't see any issues reported.  There are a lot of issues with "vcbuild chakracore jslint" after Mark's changes merged, but I don't see any in this file.

Running "node .\tools\jslint.js .\deps\chakrashim\lib\chakra_inspector.js" just complains about ever line ending being a CRLF, no other issues.